i18n/language/pluralspec.go

```diff
@@ -38,7 +45,6 @@ func (ps *PluralSpec) Plural(number interface{}) (Plural, error) {
 // getPluralSpec returns the PluralSpec that matches the longest prefix of tag.
 // It returns nil if no PluralSpec matches tag.
 func getPluralSpec(tag string) *PluralSpec {
-	tag = NormalizeTag(tag)
```

Comment:
Why did you decide to move this out of the implementation here? It looks like it is always normalized before passing in so this change seems to just bloat the diff.